iota/commands/extended/get_new_addresses.py

```diff
@@ -84,12 +94,15 @@ def __init__(self):
             | f.Max(self.MAX_SECURITY_LEVEL)
             | f.Optional(default=AddressGenerator.DEFAULT_SECURITY_LEVEL),
 
+        'checksum':  f.Type(bool) | f.Optional(default=False),
```

Comment:
👍 

`f.Optional` specifies a default value that will be used if the incoming value is `None`:
https://filters.readthedocs.io/en/latest/filters_list.html#miscellaneous-filters

This filter is occasionally important, because all the other filters will treat `None` as a passthrough value by default:
https://filters.readthedocs.io/en/latest/getting_started.html#much-ado-about-none